package.json

```diff
@@ -0,0 +1,155 @@
+    "watch-node": "DEBUG=gov-starter-server* nodemon --watch dist/ dist/server.js | bunyan -o short",
+    "watch-sass": "npm run compile-sass -- --watch",
+    "build": "npm run compile-sass && tsc && npm run copy-views",
+    "start:dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node\" \"npm run watch-sass\"",
+    "start-feature": "export $(cat feature.env) && node $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
+    "watch-node-feature": "export $(cat feature.env) && nodemon --watch dist/ $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
+    "start-feature:dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node,Sass\" -c \"yellow.bold,cyan.bold,green.bold,blue.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node-feature\" \"npm run watch-sass\"",
+    "record-build-info": "node ./bin/record-build-info",
+    "lint": "eslint . --cache --max-warnings 200",
+    "typecheck": "tsc",
+    "test": "jest --runInBand",
+    "security_audit": "npx audit-ci --config audit-ci.json",
+    "int-test": "cypress run --config video=false",
+    "int-test-ui": "cypress open",
+    "clean": "rm -rf dist build node_modules stylesheets"
+  },
+  "engines": {
+    "node": "^12.18.0",
+    "npm": "^6.4.1"
+  },
+  "jest": {
+    "preset": "ts-jest",
+    "collectCoverageFrom": [
+      "server/**/*.{ts,js,jsx,mjs}"
+    ],
+    "testMatch": [
+      "<rootDir>/(server|job)/**/?(*.)(spec|test).{ts,js,jsx,mjs}"
+    ],
+    "testEnvironment": "node",
+    "reporters": [
+      "default",
+      [
+        "jest-junit",
+        {
+          "outputDirectory": "test_results/jest/"
+        }
+      ],
+      [
+        "./node_modules/jest-html-reporter",
+        {
+          "outputPath": "test_results/unit-test-reports.html"
+        }
+      ]
+    ],
+    "moduleFileExtensions": [
+      "web.js",
+      "js",
+      "json",
+      "node",
+      "ts"
+    ]
+  },
+  "nodemonConfig": {
+    "ignore": [
+      ".circleci/*",
+      "migrations/*",
+      "node_modules/*",
+      "test/*",
+      "integration_tests/*"
+    ],
+    "delay": 2500,
+    "ext": "js,json,html,njk"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged && tsc && npm test"
+    }
+  },
+  "lint-staged": {
+    "*.{ts,js,css}": [
+      "prettier --write",
+      "eslint --fix"
+    ],
+    "*.{json}": [
+      "prettier --write"
+    ]
+  },
+  "dependencies": {
+    "@types/connect-flash": "0.0.35",
+    "agentkeepalive": "^4.1.3",
+    "applicationinsights": "^1.8.7",
+    "body-parser": "^1.19.0",
+    "bunyan": "^1.8.14",
+    "bunyan-format": "^0.2.1",
+    "compression": "^1.7.4",
+    "connect-flash": "^0.1.1",
+    "connect-redis": "^5.0.0",
+    "cookie-session": "^1.4.0",
+    "csurf": "^1.11.0",
+    "date-fns": "^2.16.1",
```

Comment:
think might be able to remove some of these?
 - govuk-elements-sass
 - date-fns

`"@types/connect-flash": "0.0.35",` should probably go in dev dependencies

Reply:
joi if the other validator stuff seems viable